domain-specific-datasets/README.md

```diff
@@ -0,0 +1,81 @@
+
+### How can you contribute?
+
+🧑🏼‍🔬 If you are a domain expert, you can contribute by sharing your expertise and collaborating with us to create domain-specific datasets. We're working with user easy to use applications that help you to define the seed data and create the dataset. We're also working on tools that help you to annotate the dataset and improve the quality of the dataset.
+
+🧑🏻‍🔧 If you are a (inspiring) Machine Learning engineer, you can setup the project and its tools. You can run the synthetic data generation pipelines. And maybe even get round to training models.
+
+
+## Project Overview
+
+### 1. Select a domain and find collaborators
+
+We start by selecting a domain and finding colaborators who can help us to create the dataset. 
+
+🧑🏼‍🔬 If you are a domain expert, you could find an ML engineer to help you to create the dataset. 
+
+🧑🏻‍🔧 If you are an ML engineer, you could find a domain expert to help you to create the dataset.
+
+🧑‍🚀 If you're both, you could start by defining the seed data and creating the dataset.
+
+### 2. Setup your project
+
+First you need o setup the project and its tools. For this, we use [this application](https://huggingface.co/spaces/argilla/domain-specific-seed). 
+
+### 3. Define the domain knowledge
+
+Next we need to get the domain expert to define the seed data. This is the data that is used to create the dataset. Once the seed data is defined, we add it to the dataset repo.
+
+![Setup the project](https://raw.githubusercontent.com/huggingface/data-is-better-together/3ac24642454764c8c7d56f0ffdd1a134c1cd37b1/domain-specific-datasets/assets/setup.png)
+
+> **Domain topics** are the topics that the domain expert wants to include in the dataset. For example, if the domain is farming, the domain topics could be "soil", "crops", "weather", etc.
+
+> **Domain description** is a description of the domain. For example, if the domain is farming, the domain description could be "Farming is the practice of cultivating crops and livestock for food, fiber, biofuel, medicinal plants, and other products used to sustain and enhance human life."
+
+> **Domain perspectives** are the perspectives that the domain expert wants to include in the dataset. For example, if the domain is farming, the domain perspectives could be "farmer", "agricultural scientist", "agricultural economist", etc.
+
+### 4. Generate the dataset
+
+Next we can move on to generating the dataset from the seed data.
+
+![Run the pipeline](https://raw.githubusercontent.com/huggingface/data-is-better-together/3ac24642454764c8c7d56f0ffdd1a134c1cd37b1/domain-specific-datasets/assets/pipeline.png)
+
```

Comment:
I'll add here the same info we have on models that people can use in the child app.